ownersort.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+find -user $USER|while read i
```

Comment:
^--^ SC2185: Some finds don't have a default path. Specify '.' explicitly.
           ^---^ SC2086: Double quote to prevent globbing and word splitting.
                       ^--^ SC2162: read without -r will mangle backslashes.